### PR TITLE
Add note on where to find ilc

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/coreclr/nativeaot/docs/optimizing.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ To aid in troubleshooting some of the most common problems related to trimming a
...
+* `<IlcInstructionSet>`: By default, the compiler targets the minimum instruction set supported by the target OS and architecture. This option allows targeting newer instruction sets for better performance. The native binary will require the instruction sets to be supported by the hardware in order to run. For example, `<IlcInstructionSet>avx2,bmi2,fma,pclmul,popcnt,aes</IlcInstructionSet>` will produce binary that takes advantage of instruction sets that are typically present on current Intel and AMD processors. Run `ilc --help` for the full list of available instruction sets. `ilc` can be executed from the NativeAOT package in your local nuget cache e.g. `C:\Users\%USERNAME%\.nuget\packages\runtime.win-x64.microsoft.dotnet.ilcompiler\7.0.0-...\tools\ilc.exe` on Windows or `~/.nuget/packages/runtime.linux-arm64.microsoft.dotnet.ilcompiler/7.0.0-.../tools/ilc` on Linux.
```
Comment (MichalStrehovsky) [inline src/coreclr/nativeaot/docs/optimizing.md:43]:
```suggestion
* `<IlcInstructionSet>`: By default, the compiler targets the minimum instruction set supported by the target OS and architecture. This option allows targeting newer instruction sets for better performance. The native binary will require the instruction sets to be supported by the hardware in order to run. For example, `<IlcInstructionSet>avx2,bmi2,fma,pclmul,popcnt,aes</IlcInstructionSet>` will produce binary that takes advantage of instruction sets that are typically present on current Intel and AMD processors. Run `ilc --help` for the full list of available instruction sets. `ilc` can be executed from the NativeAOT package in your local nuget cache e.g. `%USERPROFILE%\.nuget\packages\runtime.win-x64.microsoft.dotnet.ilcompiler\7.0.0-...\tools\ilc.exe` on Windows or `~/.nuget/packages/runtime.linux-arm64.microsoft.dotnet.ilcompiler/7.0.0-.../tools/ilc` on Linux.
```

**Comment by am11** [comment]
Should we merge src/coreclr/nativeaot/docs/ directory to top level Documentation/? (mainly to skip CI runs when modified)

**Comment by jkotas** [comment]
> Should we merge src/coreclr/nativeaot/docs/ 

This is temporary location. Most of these docs will be moved to https://github.com/dotnet/docs in next couple of months.

File: src/coreclr/nativeaot/docs/optimizing.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ To aid in troubleshooting some of the most common problems related to trimming a
...
+* `<IlcInstructionSet>`: By default, the compiler targets the minimum instruction set supported by the target OS and architecture. This option allows targeting newer instruction sets for better performance. The native binary will require the instruction sets to be supported by the hardware in order to run. For example, `<IlcInstructionSet>avx2,bmi2,fma,pclmul,popcnt,aes</IlcInstructionSet>` will produce binary that takes advantage of instruction sets that are typically present on current Intel and AMD processors. Run `ilc --help` for the full list of available instruction sets. `ilc` can be executed from the NativeAOT package in your local nuget cache e.g. `%USERPROFILE%\.nuget\packages\runtime.win-x64.microsoft.dotnet.ilcompiler\7.0.0-...\tools\ilc.exe` on Windows or `~/.nuget/packages/runtime.linux-arm64.microsoft.dotnet.ilcompiler/7.0.0-.../tools/ilc` on Linux.
```
Comment (adamsitnik) [inline src/coreclr/nativeaot/docs/optimizing.md:43]: I can't find `ilc.exe` in this location:

```cmd
C:\Projects\performance\artifacts\packages\microsoft.dotnet.ilcompiler> tree . /f
```

```log
C:\PROJECTS\PERFORMANCE\ARTIFACTS\PACKAGES\MICROSOFT.DOTNET.ILCOMPILER
└───7.0.0-preview.3.22175.4
    │   .nupkg.metadata
    │   Icon.png
    │   LICENSE.TXT
    │   microsoft.dotnet.ilcompiler.7.0.0-preview.3.22175.4.nupkg
    │   microsoft.dotnet.ilcompiler.7.0.0-preview.3.22175.4.nupkg.sha512
    │   microsoft.dotnet.ilcompiler.nuspec
    │   runtime.json
    │   THIRD-PARTY-NOTICES.TXT
    │
    ├───build
    │       BuildFrameworkNativeObjects.proj
    │       CoreRTNatVis.natvis
    │       findvcvarsall.bat
    │       Microsoft.DotNet.ILCompiler.targets
    │       Microsoft.NETCore.Native.Publish.targets
    │       Microsoft.NETCore.Native.targets
    │       Microsoft.NETCore.Native.Unix.props
    │       Microsoft.NETCore.Native.Windows.props
    │       WindowsAPIs.txt
    │
    └───tools
        └───netstandard
                ILCompiler.Build.Tasks.deps.json
                ILCompiler.Build.Tasks.dll
                System.Collections.Immutable.dll
                System.Reflection.Metadata.dll

```

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you @jkotas !

**Pull request merged**